jetson/science_bridge/src/__main__.py

```diff
@@ -249,22 +248,44 @@ def mosfet_transmit(self, channel, msg):
         pass
 
     def nav_status(self, channel, msg):  # TODO - fix make better
-        print("Received nav req")
         # Off, Done, Else
-
         struct = NavStatus.decode(msg)
+
+        if self.previous_auton_msg == struct.nav_state_name:
```

Comment:
i think the state saving here might not be super necessary, since this is probably going through teleop now i guess @CameronTressler you guys can just send a new LED command when things change on nav's end so this code base doesn't need to deal w constant publishing of NavStatus

Reply:
going to keep for now since it doesn't cause any harm to keep